Use a version of the rust crate that doesn't error out on aarch-64musl with MAP_32BIT

diff --git a/Cargo.toml b/Cargo.toml
index 31ec7ed..21ed0e6 100644
--- Cargo.toml
+++ Cargo.toml
@@ -32,6 +32,7 @@ lto = true
 
 [patch.crates-io]
 self_update = { git = "https://github.com/r-darwish/self_update", branch = "zip" }
+nix = { git = "https://github.com/nix-rust/nix", rev = "7f0fb19f21327e58faa71dace73c8b092c25c81b" } 
 
 [features]
 default = []
